ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/ElassandraCassJavaDriverPlugin.java

```diff
@@ -0,0 +1,158 @@
+package com.netflix.ndbench.plugin.cass;
```

Comment:
Should this plugin under the cassandra one? or under a separate one for Elassandra? Thoughts?